benchmark/inference/utils.py

```diff
@@ -6,13 +6,20 @@
 
 import torch_geometric.transforms as T
 from torch_geometric.datasets import OGB_MAG, Reddit
-from torch_geometric.nn.models.basic_gnn import GAT, GCN, PNA, EdgeCNN
+from torch_geometric.nn.models.basic_gnn import (
+    GAT,
+    GCN,
+    PNA,
+    EdgeCNN,
+    GraphSAGE,
+)
 
 models_dict = {
     'edge_cnn': EdgeCNN,
     'gat': GAT,
     'gcn': GCN,
     'pna': PNA,
+    'graph_sage': GraphSAGE,
```

Comment:
```suggestion
    'sage': GraphSAGE,
```
nit: prefer `sage` but this is personal taste :)

Reply:
Done.